spec/views/lists/_list.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "lists/_list" do
+  context "when the list is empty" do
+    it "shows an appropriate message" do
+      list = instance_double(List, empty?: true, name: "")
```

Comment:
I'm often frustrated that I can use plain doubles in some places, but need `build_stubbed` in others because of how Rails queries the object.